Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -110,6 +110,10 @@ YUI.add('ez-platformuiapp', function (Y) {
             contentTypeEditServerSideView: {
                 type: Y.eZ.ContentTypeEditServerSideView,
             },
+            testOneView: {
```

Comment:
Na razie może być to zrobione w ten sposób. Jednakże, sposób deklaracji widoku, do którego prowadzi routing, w eZ Studio się trochę różni. Tam używamy pluginów. Pokażę Ci przy okazji.